server/datastore/mysql/migrations/tables/20241209164540_AddSecretVariablesTable.go

```diff
@@ -0,0 +1,33 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20241209164540, Down_20241209164540)
+}
+
+func Up_20241209164540(tx *sql.Tx) error {
+
+	if tableExists(tx, "secret_variables") {
+		return nil
+	}
+
+	_, err := tx.Exec(`
+		CREATE TABLE secret_variables (
+		id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+		name VARCHAR(255) NOT NULL,
+		value BLOB NOT NULL,
+		created_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6),
+		updated_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6),
+		PRIMARY KEY (id),
+		CONSTRAINT idx_name UNIQUE (name)
```

Comment:
We usually follow a `idx_<table_name>_<col_names>` convention for indices, so in this case `idx_secret_variables_name`.

Reply:
Agree